Task_One/Firest_task.rb

```diff
@@ -0,0 +1,20 @@
+OPTIMAL = 110
+
+class IdealWeight
+  
+  def ideal_wright(name,height)
+    if height.to_i - OPTIMAL <= 0
```

Comment:
move `height.to_i - OPTIMAL` to a method

Reply:
move magic numbers to a constant

Reply:
I added new method calculate_ideal_weight
and corected magic numbers